collections/_article/dev-snapshot-godot-4-3-beta-1.md

```diff
@@ -123,6 +123,14 @@ This change however means that the scene format changed in a way that can’t be
 
 Finally, we also changed the name of the Editor Settings config file to make it specific to each Godot minor version ([GH-90875](https://github.com/godotengine/godot/pull/90875)). This avoids losing configuration when going back and forth between slightly incompatible Godot branches. The first time you use a new Godot minor branch (e.g. 4.3), it will port settings from the previous version (e.g. 4.2), but from there on the two config files stay separate.
 
+### Audio
```

Comment:
The new Audio section should be added to the table of contents under Highlights.